lib/trento/domain/cluster/events/cluster_rolled_up.ex

```diff
@@ -1,36 +1,15 @@
 defmodule Trento.Domain.Events.ClusterRolledUp do
   @moduledoc """
-  This event is emitted when a cluster is rolled up and its stream is reset.
+  This event is emitted when a cluster is rolled up and its stream is archived.
+  It contains the snapshot of the cluster aggregate that will be used to restore the aggregate state.
```

Comment:
As always, I would try to use the present term in docs.
`..aggregate that will be used` -> `..aggregate that is used to`

Reply:
intentionally in future form, because it is something that happens eventually